### DISCUSSION TITLE
"Save/Discard changes" prompt after updating program, despite no changes made.

### DISCUSSION THREAD

**Discussion by RageModeGamingYT** [Rem0o/FanControl.Releases#2193] [General]
After updating to a new version from the popup, I go to click the red X (to close the window after the program updates and reopens) and making NO changes, yet I am asked to save or discard changes. WHAT CHANGES?!? I have made no changes. Is this a bug, or does updating the program somehow change some settings?

**Comment by Rem0o** [comment]
The update brings a new setting to be serialized, hence the prompt. Just hit save and move on.

**Reply by RageModeGamingYT** [reply]
Thanks for the reply, but this should be automated to avoid confusing people who, like me, are hesitant when it comes to blindly saving unknown changes.
